src/Uno.UI.Runtime.Skia.Wpf/Extensions/Networking/Connectivity/ConnectionProfileExtension.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Runtime.InteropServices;
+using Windows.Networking.Connectivity;
+
+namespace Uno.Extensions.Networking.Connectivity
+{
+	internal class ConnectionProfileExtension : IConnectionProfileExtension
+	{
+		private readonly object _connectionProfile;
+		public ConnectionProfileExtension(object owner)
+		{
+			if (!(owner is ConnectionProfile connectionProfile))
+				throw new InvalidOperationException($"Owner of {nameof(ConnectionProfileExtension)} must be a {nameof(ConnectionProfile)}.");
```

Comment:
```suggestion
			if (!(owner is ConnectionProfile connectionProfile))
			{
				throw new InvalidOperationException($"Owner of {nameof(ConnectionProfileExtension)} must be a {nameof(ConnectionProfile)}.");
			}
```

Reply:
Easy to fix, I'll update this on my next code push

I wasn't exactly sure about the coding styles and I couldn't find a document on the contributing guide in GitHub or the documentation in Uno Docs. Is there a style guide somewhere I can reference?

Reply:
There it is: https://github.com/unoplatform/uno/blob/master/doc/articles/contributing/guidelines/code-style.md